.github/workflows/maven-release.yml

```diff
@@ -1,34 +1,63 @@
-name: Publish package to the Maven Central Repository
+name: Publish Release
+
 on:
-  push:
-    branches:
-      - main
+  ## Manual Workflow trigger
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'tag/version'
+        required: true        
+
+      action_tag:
+        description: 'Create tag? ("no" to skip)'
+        required: true
+        default: 'yes'
+
 jobs:
   publish:
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@v2
```

Comment:
Can you add a similar step to verify the version numbers before release? 
https://github.com/adobe/aepsdk-userprofile-android/blob/dev-v3.0.0/.github/workflows/maven-release.yml#L32